package-lock.json

```diff
@@ -15,6 +15,7 @@
       },
       "devDependencies": {
         "@biomejs/biome": "2.2.2",
+        "@slack/cli-hooks": "^1.2.1",
```

Comment:
suggestion: I think the `package-lock.json` was accidentally committed. We should not include it since it doesn't match the `package.json` dependencies.